templates/base/scripts/start_liferay.sh

```diff
@@ -5,6 +5,12 @@ function main {
 	echo "[LIFERAY] Starting ${LIFERAY_PRODUCT_NAME}. To stop the container with CTRL-C, run this container with the option \"-it\"."
 	echo ""
 
+	if [ "${LIFERAY_THREAD_DUMP_PROBE_ENABLED}" == "true" ]
```

Comment:
Itt ennek a blocknak mi az értelme? Nem értettem teljesen.

Reply:
Be kell állítani az LCP.json-t is, hogy rendesen működjön a dolog. Ez azért van itt, hogy egy sima docker image upgrade esetében ne fusson feleslegesen meg a kód.